frontend/src/index.html

```diff
@@ -31,7 +31,17 @@
     <div id="app" class="container-fluid p-5 min-vh-100 overflow-auto">
     </div>
     <!-- 게임 캔버스 -->
-    <canvas id="gameCanvas" tabindex="0" class="d-none h-100 w-100"></canvas>
+    <div id="gameCanvasContainer" style="position: relative;">
```

Comment:
```suggestion
    <div id="gameCanvasContainer" class="h-100 w-100" style="position: relative;">
```
지금 맵 아래가 비어 보여서 컨테이너에도 w, h 설정해주면 좋을 것 같아요!